packages/luna/src/components/Calendar/v2/WeekDays.tsx

```diff
@@ -0,0 +1,75 @@
+import { range } from "lodash";
+import React, { useMemo } from "react";
+import { DAYS_IN_WEEK } from "@/components/Calendar/v2/constants";
+import { Dayjs } from "dayjs";
+import cn from "classnames";
+import dayjs from "@/lib/dayjs";
+import { Typography } from "@/components/Typography";
+import { DayView } from "@/components/Calendar/v2/types";
+import { HOURS_IN_DAY } from "@litespace/sol/constants";
+
+export const WeekDays: React.FC<{
+  day: Dayjs;
+  DayView?: DayView;
+}> = ({ day, DayView }) => {
+  const week = useMemo(() => {
+    const weekStart = day.startOf("week");
+    return range(DAYS_IN_WEEK).map((day) => {
+      const dayStart = weekStart.add(day, "day").startOf("day");
+      const hours = range(HOURS_IN_DAY - 1).map((hour) =>
+        dayStart.add(hour, "hour")
+      );
+      return { day: dayStart, hours };
+    });
+  }, [day]);
+
+  return (
+    <>
+      {week.map(({ day, hours }, idx) => {
+        const today = day.isSame(dayjs(), "day");
+        const lastDay = week.length - 1 === idx;
+        return (
+          <div
+            className="tw-border-r tw-border-natural-300"
+            key={day.toISOString()}
+          >
+            <div
+              className={cn(
+                "tw-flex tw-items-center tw-justify-center tw-px-3 tw-h-16 tw-border-b tw-border-natural-300",
+                today ? "tw-bg-brand-100" : "tw-bg-natural-50",
+                lastDay && "tw-rounded-tl-3xl"
+              )}
+            >
+              <Typography
+                element="body"
+                weight="semibold"
+                className="tw-text-brand-700"
+              >
+                {day.format("dddd D")}
+              </Typography>
+            </div>
+
+            <div className="tw-absolute tw-w-full">
+              {hours.map((hour) => {
+                const lastHour = hour.hour() === 23;
+                return (
+                  <div
+                    key={hour.toISOString()}
+                    className={cn(
+                      "tw-h-[5.5rem] tw-border-natural-300",
+                      !lastHour && "tw-border-b"
+                    )}
+                  ></div>
```

Comment:
Self closing tags are prefared if no content. 

```suggestion
                  />
```